### PR TITLE
Add description for TCP WMI

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
This PR adds additional information for TCP metrics collected with WMI.

##### Test Plan
Take a look on descriptions.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Review by ilyam8 — CHANGES_REQUESTED** [review]
Consider not using code blocks, they very stand out and make the dashboard less usable.

Why did you choose not to use the exact wording from https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.tcp.md#metrics?

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -4200,6 +4200,34 @@ netdataDashboard.context = {
...
+        info: 'Connections with state transition from `LISTEN` to `SYN-RCVD`.'
+    },
+    'wmi.tcp_segments_received': {
+        info: 'Number of segmentts received (successful or error).'
+    },
+    'wmi.tcp_segments_retransmitted': {
+        info: 'Segments transmitted that has segments with bytes already transmitted.'
+    },
+    'wmi.tcp_segments_sent': {
+        info: 'Segments sent excluiding segments that contains "only" retransmitted bytes.'
```
Comment (shyamvalsan) [inline web/gui/dashboard_info.js:4228]: typo, excluding

File: web/gui/dashboard_info.js [hunk 2]
```diff
@@ -4200,6 +4200,34 @@ netdataDashboard.context = {
...
+        info: 'Connections with state `ESTABLISHED` or `CLOSE-WAIT`.'
+    },
+    'wmi.tcp_conns_resets': {
+        info: 'Connections with state transition from `LISTEN` to `SYN-RCVD`.'
+    },
+    'wmi.tcp_segments_received': {
+        info: 'Number of segmentts received (successful or error).'
+    },
+    'wmi.tcp_segments_retransmitted': {
+        info: 'Segments transmitted that has segments with bytes already transmitted.'
```
Comment (shyamvalsan) [inline web/gui/dashboard_info.js:4225]: has -> have

**Comment by shyamvalsan** [comment]
> Why did you choose not to use the exact wording from https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.tcp.md#metrics?

Yes, we should stick with the exact wording if possible.

**Comment by ilyam8** [comment]
The actual source is [windows documentation](https://learn.microsoft.com/en-us/previous-versions//aa394341(v=vs.85)?redirectedfrom=MSDN).

**Comment by thiagoftsm** [comment]
> The actual source is [windows documentation](https://learn.microsoft.com/en-us/previous-versions//aa394341(v=vs.85)?redirectedfrom=MSDN).

Ok, I will use the windows documentation given the link for more details.

File: web/gui/dashboard_info.js [hunk 3]
```diff
@@ -4200,6 +4200,34 @@ netdataDashboard.context = {
         info: 'Requests for which a storage exception was encountered.'
     },
 
+    // ------------------------------------------------------------------------
+    // WMI
+
+    'wmi.tcp_conns_active': {
+        info: 'Number of times `TCP` connections have made a direct transition from the `CLOSED` state to the `SYN-SENT` state.'
```
Comment (ilyam8) [inline web/gui/dashboard_info.js:4207]: Please do not use `code blocks`

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: web/gui/dashboard_info.js (hunk 3 above)
Comment (thiagoftsm) [reply web/gui/dashboard_info.js]: Done

**Review by ilyam8 — APPROVED** [review]

**Review by DShreve2 — APPROVED** [review]

**Review by shyamvalsan — APPROVED** [review]

**Pull request merged**